### DISCUSSION TITLE
How can I use outline without professionals logins

### DISCUSSION THREAD

**Discussion by boul2gom** [outline/outline#3367] [Self Hosting] [answered]
Hello ^^
I recently setup an Outline instance on my server. But I dont have any Gitlab, Azure or Google Workspace credentials. So I just want to know if I can login with classic credentials, or custom OIDC server, because the d'oc about this is empty. And please how can i allow everyone to access to my knowledge base ? Like public docs.
Thanks

**Comment by emtlovell** [comment]
you can use KeyCloak, and there is several articles on how to do it, ive linked one below. 
https://blog.gurucomputing.com.au/doing-more-with-docker/deploying-outline-wiki/

**Comment by Limezy** [comment, marked as answer]
I personally use Dex (dex.io) which is lighter and simpler than KeyCloak.
If you want to allow everyone to access your knowledgebase, you can use [my "visitor button" snippet here](https://github.com/outline/outline/discussions/3143) but you will have to rebuild Outline for it to be effective
